helm/amazon-ec2-metadata-mock/values.yaml

```diff
@@ -58,8 +58,8 @@ aemm:
   mockDelaySec: 0
   imdsv2: false
   spotItn:
```

Comment:
we shd probably change `spotItn` to `spot` as well?

Reply:
Do you know if these values get translated to env vars? If they do, then I think we have to leave it as spotItn for now because all the configs and code refs still refer to 'spot' as 'spotItn'.

Changing to 'spot' was more of a front-facing feature for better cx, updating 'spotItn' to 'spot' throughout code is a lot bigger refactor

Reply:
gotcha..  👍